### PR TITLE
add two APIs

### PR DESCRIPTION
**Pull request by byronz**

### CODE REVIEW

File: dash/testing/browser.py [hunk 1]
```diff
@@ -429,6 +433,32 @@ def clear_input(self, elem):
             .send_keys(Keys.DELETE)
         ).perform()
 
+    def zoom_in_graph_by_ratio(
+        self, elem, start_point_ratio=0.5, zoom_ratio=0.2, compare=True
```
Comment (alexcjohnson) [inline dash/testing/browser.py]: I'd call `start_point_ratio` -> `start_fraction`. But then it's not quite clear to me what `zoom_ratio` means - is it relative to the top left corner of the element like the starting point is (in which case I'd call it `end_fraction`)? Or is it relative to the starting point (in which case I'd call it `zoom_box_fraction`)? Regardless a short description of each arg, (including `compare`) would be good.

For simple "did it do something" tests on single-subplot graphs, this is fine. If anyone wants to do something more specific or precise though, we could also accept a 2-tuple for each one so you can separately specify x and y of each point.

Comment (byronz) [reply dash/testing/browser.py]: the zoom ratio controls the rectangle size when you click and drag to zoom out, the start point decides the left-top point of the rectangle

Comment (alexcjohnson) [reply dash/testing/browser.py]: OK good, so let's call them `start_fraction` and `zoom_box_fraction`. At least to me, `ratio` doesn't make it so clear what the denominator is, but `fraction` implies "fraction of the size of the element (that's the first arg)"

**Comment by byronz** [comment]
hmm, the approval disappear after update

**Comment by alexcjohnson** [comment]
:dancer:

**Pull request merged**